git_machete/cmd.py

```diff
@@ -3050,15 +3062,17 @@ def usage(c: str = None) -> None:
                       block-cancel-order
                           change-table
                               drop-location-type
+                          add-notification
             </dim>
             And now let's assume that `adjust-reads-prec` and later `block-cancel-order` were merged to develop.
             After running `git machete slide-out adjust-reads-prec block-cancel-order` the tree will be reduced to:
             <dim>
               develop
                   change-table
                       drop-location-type
+                  add-notification
             </dim>
-            and `change-table` will be rebased onto develop (fork point for this rebase is configurable, see `-d` option below).
+            and `change-table` and `add-notification` will be rebased onto develop (fork point for this rebase is configurable, see `-d` option below).
```

Comment:
Whooops... `--down-fork-point` option is tricky in case there's more than one downstream branch hmmm

Reply:
Of course it can happen that all N downstream branches have the same fork point (e.g. if they're all in sync with their parent)... but in general each of them can have a different one.

Reply:
Actually, _market research_: since I'm not an avid user of `slide-out` myself (I'm sliding out during `traverse`, or via GUI from git-machete-intellij-plugin)... do you remember yourself using `--down-fork-point` frequently?

Reply:
Maybe for clarity we could just forbid using `--down-fork-point` when there's >1 downstream branch? 🤔

Reply:
Oh, I've never actually used `--down-fork-point` and wasn't aware that it was an option.
Any time the fork-point isn't properly inferred I've already configured fork-point overrides for the branch.
I'd consider this "best practices", as it's good to verify via `git machete status -L` if there are any questions on fork points.

With this in mind, I think `--down-fork-point` seems vaguely unsafe to me.
If I'd my druthers I'd probably just remove it and enforce that fork point overrides are configured per-branch with the existing `fork-point` command before running slide out.

Given that it's possible for branches to have a common fork-point, I don't _think_ we need to disable `--down-fork-point`. 
Perhaps just print a warning that this is a potentially unsafe operation.
Heck, maybe even just deprecate `--down-fork-point` unless you know anyone who's _actually_ using it?
I could imagine the deprecation message linking back to an issue for folks to chime in if they're using it.

Reply:
> Any time the fork-point isn't properly inferred I've already configured fork-point overrides for the branch.

Yeah, you're right, I'm actually doing that myself, `--fork-point` options of `reapply` and `update`, as well as `--down-fork-point` option of `slide-out` are a relic of the times when fork point override wasn't available yet ;)

> With this in mind, I think --down-fork-point seems vaguely unsafe to me.

I agree, even more so now that you point that out 😅

> Given that it's possible for branches to have a common fork-point, I don't think we need to disable --down-fork-point.

Ok, let's leave the option as is, it's obscure but still _theoretically useful_ so I think it can stay as is. 

So maybe so that you don't spend more effort than it's really worth: what if you just rephrase the help for `--down-fork-point` option in `slide-out`'s help so that it reflects the new reality correctly?

Reply:
Done, updated existing doc and added reference to `git machete fork-point` as an preferred alternative.